README.rst

```diff
@@ -1396,6 +1396,9 @@ fixed but without actually modifying them:
 
     $ php php-cs-fixer.phar fix /path/to/code --dry-run
 
+Config File
```

Comment:
file shall be lowercased

Reply:
Actually I'm not sure: https://en.wikipedia.org/wiki/Capitalization#Titles

Reply:
it's not a title, it's header.
even in link you provided, there are `By context`, `By name of style`, `Sentence case` and so

Reply:
I thought this should be applied to headers as well, ok then :+1: